robots_101/programme_structure.md

```diff
@@ -70,10 +70,21 @@ as volunteers able to help with the kits and hands-on guidance with your robot.
 ## Virtual League
 
 The Virtual League is often the first chance you'll have to test your strategies
-against other teams. You'll [submit][code-submitter] your robot code for running
-against other teams in the [simulator][simulator]. The matches are livestreamed
-on our [YouTube Channel][youtube] and you'll get a copy of your robot's logs
-afterwards so you can continue to improve it.
+against other teams. This uses the same [simulator][simulator] used for the
+pre-Kickstart activities and the includes a [simulated robot][simulated-robot]
```

Comment:
It's not the same simulator as the pre-kickstart activities. It's the version released at kickstart.

Reply:
Hrm, technically it uses the same _simulator_ though it may not use the _simulated world_. This could probably be clarified though.

Reply:
561cbf4e7458685fd3f30e1b50afa520a5638d7f